packages/block-editor/src/components/block-list/use-block-props/use-firefox-draggable-compatibility.js

```diff
@@ -13,13 +75,9 @@ import { useRefEffect } from '@wordpress/compose';
  */
 export function useFirefoxDraggableCompatibility() {
 	return useRefEffect( ( node ) => {
-		function onDown( event ) {
-			node.draggable = ! event.target.isContentEditable;
```

Comment:
Notice that we're currently setting _all_ blocks to be draggable whenever a non editable element is clicked! Also, we're constantly setting draggable true/false for all blocks on click.